tests/unit/PlaceholderComponent/placeholder_color.spec.js

```diff
@@ -0,0 +1,30 @@
+import {mount} from "@vue/test-utils";
+import CldImage from "../../../src/components/CldImage/CldImage";
+import CldPlaceholder from "../../../src/components/CldPlaceholder/CldPlaceholder";
+import Vue from "vue";
+
+describe("CldPlaceholder", () => {
+  it("Should remove placeholder once CldImage is loaded", async () => {
+    let wrapper = mount(
+      {
+        template: `
+          <cld-image cloudName="demo" publicId="face_top">
+            <cld-placeholder type="predominant-color"></cld-placeholder>
+          </cld-image>
+        `
```

Comment:
What about testing the hasWidth scenario (the 1 pixel trick) ?

Reply:
Added